Scripts/DCS-BIOS/lib/common/Functions.lua

```diff
@@ -6,7 +6,27 @@ function ScriptPath()
     return str:match("(.*/)")
  end
 
+ --- @func Takes a string and checks for nil, returns 1 or 0
+ --- @param str string
+ --- @return integer
+ function Nil_state_to_int_flag(str)
+	if str ~= nil then
+		return 1
+	else
+		return 0
+	end
```

Comment:
is this used outside the apache? If not we should probably just keep it local to there for now. Though... it might have some utility in the F18 🤔 
```suggestion
	return str and 1 or 0
```

Reply:
fixed!